pkg/reconciler/cronjobsource/resources/labels.go

```diff
@@ -25,7 +25,7 @@ const (
 // Labels are the labels attached to all resources based on a CronJobSource.
 func Labels(name string) map[string]string {
 	return map[string]string{
-		"sources.eventing.knative.dev/cronJobSource": name,
-		"eventing.knative.dev/source":                controllerAgentName,
+		"eventing.knative.dev/source":     controllerAgentName,
+		"eventing.knative.dev/sourceName": name,
```

Comment:
this now "matches" what we have on the `ApiServersource`.

Discussion: `sources.eventing....` VS `eventing.knative.dev/source` ?

Reply:
Is there any normative set of labels to be expected by Source CRDs and if not, should such a set be added to the source's spec just under discussion ?